service/controller/v23/templates/cloudconfig/format_log_volume.go

```diff
@@ -7,7 +7,7 @@ Before=docker.service var-log.mount
 
 [Service]
 Type=oneshot
-ExecStart=/bin/bash -c "[ -e "/dev/xvdf" ] && /usr/sbin/mkfs.ext4 -f /dev/xvdf -L log"
+ExecStart=/bin/bash -c "[ -e "/dev/xvdf" ] && /usr/sbin/mkfs.ext4 -L log /dev/xvdf"
```

Comment:
if you don't use `-f` than after reboot of node this unit will fail, we either need condition that will skip formation or use force

Reply:
`-f` is not supported

```
Starting Formats EBS volume for log...
Feb 22 10:44:55 ip-10-1-0-53 systemd[1]: format-var-log.service: Main process exited, code=exited, status=1/FAILURE
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]: /usr/sbin/mkfs.ext4: invalid option -- 'f'
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]: Usage: mkfs.ext4 [-c|-l filename] [-b block-size] [-C cluster-size]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-i bytes-per-inode] [-I inode-size] [-J journal-options]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-G flex-group-size] [-N number-of-inodes] [-d root-directory]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-m reserved-blocks-percentage] [-o creator-os]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-g blocks-per-group] [-L volume-label] [-M last-mounted-directory]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-O feature[,...]] [-r fs-revision] [-E extended-option[,...]]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-t fs-type] [-T usage-type ] [-U UUID] [-e errors_behavior][-z undo_file]
Feb 22 10:44:55 ip-10-1-0-53 bash[30212]:         [-jnqvDFSV] device [blocks-count]
Feb 22 10:44:55 ip-10-1-0-53 systemd[1]: format-var-log.service: Failed with result 'exit-code'.
Feb 22 10:44:55 ip-10-1-0-53 systemd[1]: Failed to start Formats EBS volume for log.
```

this was the reason why it was failing

Reply:
I think it's `-F`: https://linux.die.net/man/8/mkfs.ext4

> Force mke2fs to create a filesystem, even if the specified device is not a partition on a block special device, or if other parameters do not make sense. In order to force mke2fs to create a filesystem even if the filesystem appears to be in use or is mounted (a truly dangerous thing to do), this option must be specified twice.

Reply:
I'll try that but it does not appear as a flag in https://github.com/giantswarm/aws-operator/pull/1412#discussion_r259311839